src/server/database_provider.py

```diff
@@ -1,24 +1,21 @@
-import sqlite3
 import os
+import sqlite3
 
 from . import PATHS
 
 
-def getDatabaseConnection():
```

Comment:
I will reinstate this function back, but note that it should be wrapped into a class, because in this file there are two things happening: connection and initialization of database.

Also, what will happen when connection fails? There's no safety check.

Reply:
Bezpieczeństwo w przypadku błędu połączenia może być realizowane w taki sam sposób, jak bez tej funkcji. W końcu błędy wychodzą na zewnątrz, to pierwszego momentu, gdzie są łapane.
Poprawię kod w osobnym pr, tak by w module była tylko jedna publiczna funkcja.